plugins.go

```diff
@@ -85,8 +54,31 @@ func findAndLoadPlugins(o *Options) error {
 			return fmt.Errorf("open plugin %s from %s: %s\n", name, path, err)
 		}
 
+		conf, err := readPluginConfig(path)
+		if err != nil {
+			return fmt.Errorf("failed to read config for %s: %s", path, err)
+		}
+
+		if sym, err := mod.Lookup("InitPlugin"); err == nil {
+			fltr, pred, dc, err := loadMultiPlugin(sym, path, conf)
+			if err != nil {
+				return fmt.Errorf("filter plugin %s returned: %s\n", path, err)
+			}
+			if fltr != nil {
+				o.CustomFilters = append(o.CustomFilters, fltr)
+			}
+			if pred != nil {
+				o.CustomPredicates = append(o.CustomPredicates, pred)
+			}
+			if dc != nil {
+				o.CustomDataClients = append(o.CustomDataClients, dc)
+			}
+			fmt.Printf("mutlitype plugin %s loaded from %s (filter: %t, predicate: %t, dataclient: %t)\n",
```

Comment:
We follow the convention of printing log messages to the stderr. Please, change 'fmt.Println' to log.Println, using logrus. (Logrus allows the user to override the output.)